NEWS.md

```diff
@@ -1,8 +1,9 @@
-# worldfootballR 0.6.2.1000
+# worldfootballR (development version)
```

Comment:
this should prevent a merge conflict i think

Reply:
Almost, however there are still some merge conflicts. I’m not in front of my laptop for a while - are you able to fix these?

Reply:
alright it should be resolved now